app/Svg/Builder.hs

```diff
@@ -216,5 +233,5 @@ transposeMatrix3x3 _ = error "Matrix must be 3x3"
 
 -- Parse transform back from the format stored in the database
 listToMatrix :: [Double] -> Matrix
-listToMatrix [a, b, c, d, e, f] = [[a, c, e], [b, d, f], [0, 0, 1]]
+listToMatrix [a, b, c, d, e, f] = [[a, b, e], [c, d, f], [0, 0, 1]]
```

Comment:
On the other hand, this change seems suspicious. The SVG `transform` `matrix` version stores the six numbers in column-major order (https://developer.mozilla.org/en-US/docs/Web/SVG/Attribute/transform#matrix) so assuming these six numbers in the database are in the original order of the `matrix` attribute, the original code is correct.